docs/pipeline/rules/headerscollection.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "Headers Collection"
+description: "The \"Headers Collection\" rule can be used to add span attributes by collecting HTTP Headers data."
+sidebarTitle: "Headers Collection"
+icon: "webhook"
+---
+
+<Info>
+  This Instrumentation Rule is currently only available with the Odigos **Enterprise** plan.<br />
+  [Contact us](https://odigos.io/) for more information.
+</Info>
+
+## Considerations
+
+<Warning>
```

Comment:
consider adding a warning that http headers might contain PII data (like cookies or api keys) thus it is recommended to review each requested header before configuring it here to avoid sensitive information.